### PR TITLE
DDF-2857 Move itests to integration-test phase

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Moves the itests to the integration-test phase using the failsafe plugin
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@tbatie @brendan-hofmann 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Build](https://github.com/orgs/codice/teams/build)
[Test](https://github.com/orgs/codice/teams/test)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@figliold
@lessarderic

#### How should this be tested? (List steps with links to updated documentation)
Verify itests don't run with maven clean test
Verify itests pass with maven clean install
#### What are the relevant tickets?
[DDF-2857](https://codice.atlassian.net/browse/DDF-2857)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [X] Update / Add Integration Tests

### CODE REVIEW

**Comment by clockard** [comment]
build now

File: distribution/test/itests/test-itests-ddf/pom.xml [hunk 1]
```diff
@@ -228,21 +228,78 @@
...
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
+            <version>1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```
Comment (shaundmorris) [inline distribution/test/itests/test-itests-ddf/pom.xml]: Should we use a property for junit or do we need this specific version?

Comment (clockard) [inline distribution/test/itests/test-itests-ddf/pom.xml]: Good catch. Fixed

File: distribution/test/itests/test-itests-ddf/pom.xml [hunk 2]
```diff
@@ -228,21 +228,78 @@
...
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.8.1</version>
                 <configuration>
+                    <skipTests>true</skipTests>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <!--Using version 2.17 instead of the latest (2.19.1) because everything after 2.17 has-->
```
Comment (tbatie) [inline distribution/test/itests/test-itests-ddf/pom.xml]: Out of curiosity, what type of exceptions did you see upgrading this plugin?

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java [hunk 3]
```diff
@@ -433,7 +434,8 @@ private static Integer findPortNumber(Integer portToTry) {
                 // increase timeout for CI environment
                 systemTimeout(TimeUnit.MINUTES.toMillis(10)),
                 when(Boolean.getBoolean("keepRuntimeFolder")).useOptions(keepRuntimeFolder()),
-                cleanCaches(true));
+                cleanCaches(true),
+                configureConsole().ignoreLocalConsole());
```
Comment (tbatie) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java]: What exactly is this doing to the console and why is it needed? I couldn't find good documentation on it.

File: distribution/test/itests/test-itests-ddf/pom.xml (hunk 2 above)
Comment (clockard) [inline distribution/test/itests/test-itests-ddf/pom.xml]: I was seeing errors with the pax exam probe not connecting.

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java (hunk 3 above)
Comment (clockard) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java]: This was from an example I found when trying to figure out other issues. I retested without it and it is not needed. Removed the ignoreLocalConsole

**Comment by clockard** [comment]
build now

**Pull request merged**